framework/src/timesteppers/FunctionDT.C

```diff
@@ -20,7 +20,7 @@ template<>
 InputParameters validParams<FunctionDT>()
 {
   InputParameters params = validParams<TimeStepper>();
-  params.addParam<std::vector<Real> >("time_t", "The values of t");
+  params.addRequiredParam<std::vector<Real> >("time_t", "The values of t");
   params.addParam<std::vector<Real> >("time_dt", "The values of dt");
```

Comment:
I see it in your input file, by specifying `time_t`, one would prescribe dt for each step. Is `FunctionDT` the right place for this?